WTWSMS/localisation/WtWSMS_bureaucratic.csv

```diff
@@ -0,0 +1,9 @@
+# Bureaucratic and Sub-Roman title prefixes;;;;;;;;;;;;x
+bureaucratic_duchy_greek;Eparchy;Éparchie;Theme;;Theme;;;;;;;;;x
+subroman_duchy_greek;Eparchy;Éparchie;Theme;;Theme;;;;;;;;;x
+bureaucratic_duchy_of_greek;Eparchy of;Éparchie de;Theme von;;Theme de;;;;;;;;;x
+subroman_duchy_of_greek;Eparchy of;Éparchie de;Theme von;;Theme de;;;;;;;;;x
+bureaucratic_duchy_griko;Eparchy;Éparchie;Theme;;Theme;;;;;;;;;x
+subroman_duchy_griko;Eparchy;Éparchie;Theme;;Theme;;;;;;;;;x
+bureaucratic_duchy_of_griko;Eparchy of;Éparchie de;Theme von;;Theme de;;;;;;;;;x
+subroman_duchy_of_griko;Eparchy of;Éparchie de;Theme von;;Theme de;;;;;;;;;x
```

Comment:
I would still say this does not need to be a separate file, in my view it is easier to centralise it in a single large file on titles such as the one we already have. Or do you plan on adding more of these?